qwikidata/json_dump.py

```diff
@@ -30,13 +31,13 @@ def __init__(self, filename: str) -> None:
 
         if filename.endswith(".json"):
             self.basename, _ = os.path.splitext(filename)
-        elif filename.endswith(".json.bz2"):
+        elif filename.endswith((".json.bz2", "json.gz")):
```

Comment:
can we include the "." in front of "json.gz" for consistency with the other checks?